lldb/test/Shell/SwiftREPL/DictBridging.test

```diff
@@ -2,7 +2,6 @@
 // Test formatters on bridged dictionaries in the REPL.
 // REQUIRES: system-darwin
 // REQUIRES: swift
-// REQUIRES: rdar73216083
```

Comment:
does `SetBridging.test` need to be updated too?

Reply:
Done.